tests/sentry/api/endpoints/test_accept_organization_invite.py

```diff
@@ -14,14 +14,36 @@
     OrganizationMember,
 )
 from sentry.testutils import TestCase
+from sentry.testutils.silo import region_silo_test
 
 
+@region_silo_test
 class AcceptInviteTest(TestCase):
     def setUp(self):
         super().setUp()
         self.organization = self.create_organization(owner=self.create_user("foo@example.com"))
         self.user = self.create_user("bar@example.com")
 
+    def _get_paths(self, args):
+        return (
+            reverse("sentry-api-0-accept-organization-invite", args=args),
+            reverse(
+                "sentry-api-0-accept-organization-invite-with-org",
+                args=[self.organization.slug] + args,
+            ),
+        )
```

Comment:
With membership invitations expiring in a non-infinite time we should be able to remove the non-organization scoped web URL and API endpoint faster than other endpoints.